Kernel/Arch/x86/common/Processor.cpp

```diff
@@ -551,10 +551,6 @@ UNMAP_AFTER_INIT void Processor::cpu_setup()
         write_cr4(read_cr4() | 0x800);
     }
 
-    if (has_feature(CPUFeature::TSC)) {
```

Comment:
I think this was done as some sort of security mitigation?
The commit should describe why the change is being made as well...

Reply:
Digging around in `git blame` and looking at the original commit by Andreas shows that this was done on purpose and it heavily implies a resulting security benefit, but it doesn't mention which in particular. A quick look on Wikipedia results in that `rdtsc` is frequently used to accurately time side-channel attacks.

However, this coincides with QEMU also trying to accurately time the running machine, which I will document in the commit message.